network.c

```diff
@@ -1139,6 +1139,21 @@ struct iio_context * network_create_context(const char *host)
 		ret = getaddrinfo(addr_str, port, &hints, &res);
 	} else {
 		ret = getaddrinfo(host, port, &hints, &res);
+#ifdef _WIN32
+		/* Yes, This is lame, but Windows is flakey with local addresses
+		 * WSANO_DATA = Valid name, no data record of requested type.
+		 * Normally when the host does not have the correct associated data
+		 * being resolved for. Just ask again.
+		 */
+		if (HAVE_DNS_SD && ret == WSANO_DATA) {
+			i = 0;
+			while (ret == WSANO_DATA && i < 5) {
+				Sleep(500);
```

Comment:
yeah, I tested with both, and it seemed to only need a single 500ms delay most of the time (I saw once it looped twice) - I did it to one second before so it would be the same on linux and windows if we ever wanted to do that, but this does make it more responsive...

Reply:
What about 25 times 100ms? Would it be faster?

Reply:
yes - updated.